base/strings/io.jl

```diff
@@ -250,7 +250,7 @@ print(io::IO, s::Union{String,SubString{String}}) = (write(io, s); nothing)
 """
     repr(x; context=nothing)
 
-Create a string from any value using the 2-argument `show(io, x)` function.
+Create a string representation of any value using the 2-argument `show(io, x)` function.
```

Comment:
Is it correct to go further and mention the idea of `eval(Base.Meta.parse(repr(x))) == x`?
I'm not sure whether it's strictly enforced though?
Something like this?
```suggestion
Create a string representation of any object using the 2-argument `show(io, x)` function.
In most cases `eval(Base.Meta.parse(repr(x))) == x` will hold true.
```

Reply:
The language used in the `show` docstring is "should be parseable Julia code when possible".  I like your suggestion to add that property.

I don't see why we should prefer "object" over "value".

Reply:
I thought object was a superset of value but perhaps that's not right. I'll update my suggestion above